include/mneme/storage.hpp

```diff
@@ -111,6 +118,12 @@ template <DataLayout TDataLayout, typename... Ids> class MultiStorage {
         return access_policy_t<Extent>::get(offset, from, to);
     }
 
+    void resize(std::size_t size) {
+        size_ = size;
+        allocate_policy_t::deallocate(values);
+        allocate_policy_t::allocate(values, size);
```

Comment:
Just as an idea, this could be done in a allocate_policy_t::reallocate method which could be potentially implemented more efficiently using realloc - or do the same thing as this method

Reply:
Realloc doesn't work with objects with copy constructors, so maybe it's a bad idea.